Backpack/Gradient/Classes/BPKGradient.h

```diff
@@ -156,7 +152,9 @@ NS_ASSUME_NONNULL_BEGIN
  * @param direction The desired direction.
  * @return The Skyscanner primary gradient.
  */
-+ (instancetype)primaryWithDirection:(BPKGradientDirection)direction;
++ (instancetype)primaryWithDirection:(BPKGradientDirection)direction NS_SWIFT_NAME(primaryWith(direction:));
```

Comment:
can you specify `primary(with direction:)` to better align with Swift 3.0+ naming standards? So you could call the method `primary(with: GradientDirection.up)`. (Same applies to start/endPoint(for: direction))

Reply:
Yeah agreed

Reply:
Should be `primary(direction: .up)` though?

Reply:
yeah, good point